plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -190,52 +190,66 @@ func IncrementUsage(a admission.Attributes, status *api.ResourceQuotaStatus, cli
 			}
 		}
 	}
-	// handle memory/cpu constraints, and any diff of usage based on memory/cpu on updates
-	if a.GetResource() == "pods" && (set[api.ResourceMemory] || set[api.ResourceCPU]) {
-		pod := obj.(*api.Pod)
-		deltaCPU := resourcequotacontroller.PodCPU(pod)
-		deltaMemory := resourcequotacontroller.PodMemory(pod)
-		// if this is an update, we need to find the delta cpu/memory usage from previous state
-		if a.GetOperation() == admission.Update {
-			oldPod, err := client.Pods(a.GetNamespace()).Get(pod.Name)
-			if err != nil {
-				return false, err
+
+	if a.GetResource() == "pods" {
+		for _, resourceName := range []api.ResourceName{api.ResourceMemory, api.ResourceCPU} {
+			// ignore tracking the resource if its not in the quota document
```

Comment:
I think we've been using the convention of putting comments above the line they comment, so this comment should be moved up one line.